src/modules/project/pages/projectView/views/projectActivityPanel/screens/qr/views/onchain/refund/musig.ts

```diff
@@ -0,0 +1,22 @@
+import { Transaction } from 'bitcoinjs-lib'
+import { Taptree } from 'bitcoinjs-lib/src/types'
+import { Musig, RefundDetails, TaprootUtils } from 'boltz-core'
+import { Buffer } from 'buffer'
+import { ECPairInterface } from 'ecpair'
+
+import { secp } from './helpers'
+import { generateRandomBytes } from './utils'
+
+export const createMusig = (ourKeys: ECPairInterface, theirPublicKey: Buffer) => {
```

Comment:
who's "their" and "our"? Is it Geyser and Boltz? Can you rename those vars for clarity?